website/docs/dependencies/0050-install-java-jdk.md

```diff
@@ -32,31 +32,27 @@ The following section describes the parameters used by the individual component
 
 | **Parameter** | **Required** | **Description**                                                                                                 |
 |---------------|--------------|-----------------------------------------------------------------------------------------------------------------|
```

Comment:
Thanks for updating the document. Please add 1/2 sentences about how this derives from the DependencyPackageInstallation

Reply:
Updated documentation along with installation steps